nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LogAttribute.java

```diff
@@ -123,11 +131,15 @@ protected void init(final ProcessorInitializationContext context) {
     protected String processFlowFile(final ProcessorLog logger, final DebugLevels logLevel, final FlowFile flowFile, final ProcessSession session, final ProcessContext context) {
         final Set<String> attributeKeys = getAttributesToLog(flowFile.getAttributes().keySet(), context);
         final ProcessorLog LOG = getLogger();
-
+        String logPrefix = context.getProperty(LOG_PREFIX).getValue();
         // Pretty print metadata
         final StringBuilder message = new StringBuilder();
         message.append("logging for flow file ").append(flowFile);
         message.append("\n");
+        if (logPrefix != null) {
+            message.append(logPrefix);
+            message.append(" ");
+        }
         message.append(FIFTY_DASHES);
```

Comment:
Since we are changing the output format would like to see a String.format or something similar to give a consistent width with number of dashes.